sysbrokers/IB/config/ib_config_futures.csv

```diff
@@ -320,12 +326,12 @@ MSCICHINA,M1CNX,EUREX,USD,50,1,FALSE
 MSCICHINANET,MCN.HK,HKFE,USD,50,1,FALSE
 MSCIEAFA,MXEA,NYSELIFFE,USD,50,1,FALSE
 MSCIEAFA-TOT,M1EA,CME,USD,20,1,FALSE
-MSCIEM,M1EF,EUREX,USD,50,1,FALSE
+MSCIEM,M1EF,EUREX,USD,100,1,FALSE
```

Comment:
Just double checked this and it's right.

Note that MSCI EM Net Total Return Futures also trade as M1EF on NYSELIFFE. Should we add a separate instrument for that? My impression is that volumes on EUREX and NYSELIFFE are pretty much the same for this instrument.